spec/controllers/squads_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.describe SquadsController, type: :controller do
+  let(:squad) { create(:squad) }
+  let(:valid_attr) { attributes_for(:squad) }
+  let(:invalid_attr) { attributes_for(:squad, title: '') }
+  let(:new_attr) { attributes_for(:squad, title: 'Blåtand') }
+
+  allow_user_to :manage, Squad
+
+  describe "GET #index" do
+    it "assigns" do
+      squad
+      get :index
+      assigns(:squads).should include(squad)
+    end
+  end
+
+  describe "GET #show" do
+    it "assigns the requested squad as @squad" do
+      get :show, id: squad.to_param
+      assigns(:squad).should eq(squad)
+    end
+  end
+
+  describe "GET #new" do
+    it "assigns a new squad as @squad" do
+      get :new
+      assigns(:squad).should be_a_new(Squad)
+    end
+  end
+
+  describe "GET #edit" do
+    it "assigns the requested squad as @squad" do
+      get :edit, id: squad.to_param
+      assigns(:squad).should eq(squad)
+    end
+  end
+
+  describe "POST #create" do
+    context "with valid params" do
+      it "creates a new Squad" do
+        lambda do
+          post :create, squad: valid_attr
+        end.should change(Squad, :count).by(1)
+      end
+
+      it "assigns a newly created squad as @squad" do
+        post :create, squad: valid_attr
+        assigns(:squad).should be_a(Squad)
+        assigns(:squad).should be_persisted
+      end
+
+      it "redirects to the created squad" do
+        post :create, squad: valid_attr
+        response.should redirect_to(Squad.last)
+      end
+    end
+
+    context "with invalid params" do
+      it "assigns a newly created but unsaved squad as @squad" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.